sklearn/decomposition/kernel_pca.py

```diff
@@ -77,28 +74,40 @@ class KernelPCA(BaseEstimator, TransformerMixin):
         When n_components is None, this parameter is ignored and components
         with zero eigenvalues are removed regardless.
 
-    random_state : int seed, RandomState instance, or None, default : None
+    random_state : int seed, RandomState instance, or None, default=None
         A pseudo random number generator used for the initialization of the
         residuals when eigen_solver == 'arpack'.
 
-    n_jobs : int, optional (default = 1)
+    n_jobs : int, default=1
         The number of parallel jobs to run.
-        If ``-1``, then the number of jobs is set to the number of CPU cores.
+        If `-1`, then the number of jobs is set to the number of CPU cores.
+
+    copy_X : boolean, default=True
+        If True, input X is copied and stored by the model in the `X_fit_`
+        attribute. If no further changes will be done to X, setting
+        `copy_X=False` saves memory by storing a reference.
 
     Attributes
     ----------
+    lambdas_ : array, (n_components,)
+        Eigenvalues of the centered kernel matrix in decreasing order.
+        If `n_components` and `remove_zero_eig` are not set,
+        then all values are stored.
+
+    alphas_ : array, (n_samples, n_components)
+        Eigenvectors of the centered kernel matrix. If `n_components` and
+        `remove_zero_eig` are not set, then all components are stored.
 
-    lambdas_ :
-        Eigenvalues of the centered kernel matrix
+    dual_coef_ : array, (n_samples, n_features)
+        Inverse transform matrix. If `fit_inverse_transform=False`,
+        dual_coef_=None
```

Comment:
Looking at the code I don't think this is the case: if `fit_inverse_transform` is False, this will be missing (AttributeError). Maybe the docstring should say "only present if `fit_inverse_transform=False`".